### PR TITLE
Add textarea input

### PR DESCRIPTION
**Pull request by nuno-agostinho**
Includes the functions to create new textarea input and to update an existing one. Both of these functions were adapted from the respective functions for the text input.

### CODE REVIEW

File: R/input-textarea.R [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#' }
+#' @export
+textAreaInput <- function(inputId, label, value = "", width = NULL,
+                      placeholder = NULL) {
+
+  div(class = "form-group shiny-input-container",
+      style = if (!is.null(width)) paste0("width: ", validateCssUnit(width), ";"),
+      label %AND% tags$label(label, `for` = inputId),
+      tags$textarea(id = inputId, class = "form-control", value = value,
+                    placeholder = placeholder)
```
Comment (jcheng5) [inline R/input-textarea.R]: I believe this should be:

```
tags$textarea(id = inputId, class = "form-control", placeholder = placeholder,
    paste(value, collapse = "\n"))
```

as you set the default value of textarea like this: `<textarea>default value</textarea>`

Might also want height, rows, and cols arguments (all set to NULL by default).

**Comment by jcheng5** [comment]
Thanks for this contribution. Can I trouble you to sign a CLA, if you've never signed one for RStudio? This is necessary to let us make changes to the license in the future (for example, to change to a BSD license if GPL becomes a problem, or a later GPL version). If that's OK with you, please email me at joe@rstudio.com and I'll send you the form--you can sign it and send me a cellphone pic or scan.

Thanks!

**Comment by nuno-agostinho** [comment]
Hey @jcheng5! I've already sent an email to @jjallaire with the signed [RStudio Individual Contributor Agreement](https://www.rstudio.com/wp-content/uploads/2014/06/RStudioIndividualContributorAgreement.pdf). I've forwarded to you in case that's the document you refer to!

Regarding the code, I may notice the `height` is inside the `tags$textarea` because it didn't work in the place where the `width` is defined. Also, I couldn't put the `cols` argument working. :(

**Comment by jcheng5** [comment]
Oh thanks, didn't realize you had one on file. Thanks!

**Comment by wch** [comment]
Closing this in favor of #1300.

**Pull request closed without merging**